jobrunner/cli/manifests.py

```diff
@@ -0,0 +1,94 @@
+"""
+Ops utility for backfilling manifest.json files from db
+"""
+import argparse
+
+from jobrunner.executors import local
+from jobrunner.lib import database
+from jobrunner.models import Job
+
+
+def main(workspaces=None):
+    conn = database.get_connection()
+    workspaces = [
+        w["workspace"] for w in conn.execute("SELECT DISTINCT(workspace) FROM job;")
+    ]
+
+    n_workspaces = len(workspaces)
+    for i, workspace in enumerate(workspaces):
+        print(f"workspace {i+1}/{n_workspaces}: {workspace}")
+
+        level4_dir = local.get_medium_privacy_workspace(workspace)
+
+        sentinel = level4_dir / ".manifest-backfill"
+        if sentinel.exists():
```

Comment:
Why do you need a sentinal file? Isn't the `manifest.json` itself the sentinal?

Reply:
Because manifest.json does still exist, it much neutered form, so that osrelease knows what workspace this directory is for.

Reply:
Ahha! OK, fine